### ISSUE TITLE
Volumes API

### ISSUE THREAD

**Issue by bwateratmsft** [docker-archive-public/docker.node-sdk-legacy#34]
The new Docker CLI recently added volumes support (at least in ACI contexts). Can this SDK also add support for that? I assume it would mean new protos for volumes.

**Comment by gtardif** [comment]
Yes it's in our plans, a Volume API for ACI contexts should arrive soon here

**Comment by gtardif** [comment]
cc @ulyssessouza

**Comment by bwateratmsft** [comment]
Looks like implementation in #43 is good. The Edge channel does not yet have ACI integration 0.1.18; is there something newer I can test with or should I just hold off for now?

**Comment by karolz-ms** [comment]
@gtardif said in an email:

> we just released node-sdk v0.1.18 This includes APIs for volume management (list, create, delete) on ACI. It will require the docker ACI integration 0.1.18, coming with Docker Desktop 2.4.0.0 (Desktop stable)

So sounds like we should only enable the feature if the user has Desktop 2.4.0.0 or newer
